app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -66,6 +68,28 @@ const CORE_TOP_TABS = [
       return !this.currentUser.likes_notifications_disabled;
     }
   },
+
+  class extends UserMenuTab {
+    get id() {
+      return REVIEW_QUEUE_TAB_ID;
+    }
+
+    get icon() {
+      return "flag";
+    }
+
+    get panelComponent() {
+      return "user-menu/reviewables-list";
+    }
+
+    get shouldDisplay() {
+      return this.currentUser.can_review;
+    }
+
+    get count() {
+      return this.currentUser.get("reviewable_count");
```

Comment:
we generally don't need get anymore

Reply:
Yes, I'm aware, but this is one of the rare cases where we actually need it until we refactor the `User` model to be a native class with `@tracked` properties, see https://github.com/discourse/discourse/pull/17630#discussion_r929521750.